prov/util/src/util_mem_monitor.c

```diff
@@ -192,19 +193,29 @@ int ofi_monitors_add_cache(struct ofi_mem_monitor **monitors,
 
 		if (dlist_empty(&monitor->list)) {
 			ret = monitor->start(monitor);
-			if (ret) {
-				cache->monitors[iface] = NULL;
-				goto out;
-			}
+			if (ret && ret != -FI_ENOSYS)
+				goto err;
```

Comment:
I believe we need to continue in the ENOSYS case.  I've updated this patch and opened #6097 with this change.